selfdrive/boardd/boardd.cc

```diff
@@ -115,7 +115,7 @@ bool safety_setter_thread(std::vector<Panda *> pandas) {
     return false;
   }
 
-  pandas[0]->set_safety_model(cereal::CarParams::SafetyModel::ELM327);
+  pandas[0]->set_safety_model(cereal::CarParams::SafetyModel::ALL_OUTPUT);
```

Comment:
we'll need to make a safety mode for the body since all output isn't in the release panda build. for now, you can just pass all output from the car interface to remove this hack